src/extension/features/toolkit-reports/pages/spending-by-category/component.tsx

```diff
@@ -180,40 +214,42 @@ export class SpendingByCategoryComponent extends React.Component {
       });
 
       drillDownData.push({
-        data: spendingData.get('sources').map((subCategoryData, subCategoryIndex) => ({
+        // @ts-ignore custom payload on point
```

Comment:
Prefer `@ts-expect-error` over `@ts-ignore`

Best of all would be to modify the type of `drillDownData` to accept this without having to ignore the "wrong" `.push`.